lib/src/core/message/guild_emoji.dart

```diff
@@ -102,19 +155,43 @@ class GuildEmoji extends BaseGuildEmoji implements IGuildEmoji {
   @override
   late final bool animated;
 
+  /// The name of the emoji.
+  @override
+  late final String name;
+
   @override
   bool get isPartial => false;
 
+  /// Returns cdn url to emoji
+  @override
+  String get cdnUrl => "https://cdn.discordapp.com/emojis/$id.${animated ? "gif" : "png"}";
+
+  /// The user that created this emoji, not present if [fetchCreator] has not been called
+  @override
+  late Cacheable<Snowflake, IUser>? creator;
```

Comment:
The `late` keyword here isn't necessary.

Reply:
Yup, fixed it, sorry for this mistake